packages/salesforcedx-vscode-core/src/messages/i18n.ja.ts

```diff
@@ -250,6 +250,8 @@ export const messages = {
     'sfdx-project.json ファイル (%s) をパースできませんでした。パースエラー: %s',
   sfdx_cli_not_found:
     'Salesforce CLI がインストールされていません。[%s](%s) からインストールしてください。',
```

Comment:
I updated the English error message in i18n.ts but the Japanese error message is still the old one. @sbudhirajadoc